app/models/manageiq/providers/vmware/infra_manager/metrics_capture.rb

```diff
@@ -390,6 +399,16 @@ def perf_query(params, counter_info, interval_name)
     return counters_by_mor, counter_values_by_mor_and_ts
   end
 
+  private
+
+  def log_targets
+    if targets.size == 1
+      "[#{target.class.name}], [#{target.id}], [#{target.name}]"
```

Comment:
Is target (singular) an accessor? or do you need a `target = targets.first` here?

Reply:
It's a reader from the base class, https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/base_manager/metrics_capture.rb#L4